### DISCUSSION TITLE
Adding individuals on local install

### DISCUSSION THREAD

**Discussion by karynne7** [broadinstitute/seqr#2095] [General]
We have two WES trios uploaded with ES, a project and two families. However, we cannot create individuals via the project upload or on the admin backend. We get "Unable to find matching ids for 6 individuals. The following entries will not be updated: ..." on the first, and on the backend it's requiring us to fill out many fields, including jsons for features and etc. and won't take what I've filled out. Is there a template for the backend, or some feature we haven't turned on? I am "staff" in settings and have tried the project way with xlsx and tsv, and files I've downloaded from other tickets with my IDs instead. I really don't think it's formatting. 
```
Family ID	Individual ID	Paternal ID	Maternal ID	Sex	Affected Status	Notes
SF013	326850			1	1	father-SF013FA171
SF013	326851			2	1	mother-SF013MO170
SF013	326852	326850	326851	1	2	proband-SF013PR168
SF017	326853			1	1	father-SF017FA311
SF017	326854			2	1	mother-SF017MO186
SF017	326855	326853	326854	1	2	proband-SF017PR185
```

**Comment by hanars** [comment]
You need to upload them using the "Bulk Edit Individuals" tab, not the "Bulk Edit Individuals Metadata" tab

**Reply by karynne7** [reply]
I believe that's what I have tried under the admin page. And I get errors without info here:
<img width="569" alt="Screen Shot 2021-09-02 at 10 45 06 AM" src="https://user-images.githubusercontent.com/47792504/131892057-1624492c-b988-42a2-9aa1-c3fcfccda924.png">

**Reply by hanars** [reply]
please never use the admin page for routine seqr work. As I said, you should use the "Bulk Edit Individuals" tab that appears when you click "edit Families an Individuals" on the project page

**Comment by karynne7** [comment]
Thanks, Hana. I think something is wrong with our cloud deployment because I don't see that option, and I have online and in an all on-prem deployment we had. We'll keep digging. Appreciate the feedback.

**Reply by hanars** [reply]
You need to be a superuser to see this

**Reply by karynne7** [reply]
Apparently we needed to enable case review that was not turned on on our version. Again, really appreciate the help. It's working as expected again.
